src/messageBridge.ts

```diff
@@ -0,0 +1,73 @@
+import { RuneGameCommand, RuneGameEvent } from "./types"
+
+const RUNE_MESSAGE_PREFIX = "RUNE_MSG;"
```

Comment:
Might be nice with a comment to explain the rationale behind using a prefix before the stringified JSON :)

Reply:
Thanks, good point! Added the comment: [`c73e270` (#53)](https://github.com/rune/rune-games-sdk/pull/53/commits/c73e270eda6ce3d9c97c7971b22de94c921134c3)